strimzi-container/src/main/java/io/strimzi/StrimziContainer.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi;
+
+import com.github.dockerjava.api.command.ExecCreateCmdResponse;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.github.dockerjava.api.model.ContainerNetwork;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+
+public class StrimziContainer extends GenericContainer<StrimziContainer> {
+
+    private static final Logger LOGGER = LogManager.getLogger(StrimziContainer.class);
+
+    private static final String STARTER_SCRIPT = "/testcontainers_start.sh";
+    private static final int KAFKA_PORT = 9093;
+    private static final int ZOOKEEPER_PORT = 2181;
+
+    private int kafkaExposedPort;
+    private StringBuilder advertisedListeners;
+
+    public StrimziContainer(final String version) {
+        super("strimzi/kafka:" + version);
+        super.withNetwork(Network.SHARED);
+
+        // exposing kafka port from the container
+        withExposedPorts(KAFKA_PORT);
+    }
+
+    public StrimziContainer() {
+        this("latest-kafka-2.5.0");
```

Comment:
This is tricky and would need to be maintained maually in releases etc. So I think this needs to be generated from `make release` or something like that.

Reply:
updated